packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/BarGraph/Reflow/DynamicData.2.doc.mdx

```diff
@@ -31,5 +33,10 @@ import BarGraph from '@cerner/terra-graphs/lib/components/Bar/BarGraph';
 <GraphConfig />
 
 ### Data object
-<DataObject />
+
+#### Initial Data
+<DataObject1 />
+
+#### Updated Data
+<DataObject1 />
```

Comment:
Should this be DataObject2?

Reply:
fixed: e7ec1f1a0eaf69391363e1e9ad7b706d10f7307a